src/core/globalstate.ts

```diff
@@ -45,6 +45,12 @@ export class MobXGlobals {
 	isRunningReactions = false;
 
 	/**
+	 * Current reaction (if any is running).
+	 * Reactions are run after derivations using a trampoline.
+	 */
+	currentReaction: Reaction = undefined;
```

Comment:
yeah that is how I came to it :) passing an additional param to the effect of reaction felt inconsistent with the rest of the reaction api's.